tests/assignment_comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -61,12 +123,114 @@ let f3 = (
   b //comment
 ) => {};
 
+let f4 = // Comment
+  () => {};
+
+let f5 =
+
+  // Comment
+
+  () => {}
+
+let f6 = /* comment */
+
+  // Comment
+
+  () => {}
+
+fnNumber =
+  // Comment
+  3;
+
+fnNumber =
+
+  // Comment
+
+  3;
+
+fnNumber =
+  // Comment0
+  // Comment1
+  3;
+
+fnNumber = /* comment */
+  3;
+
+fnNumber = /* comments0 */
+  /* comments1 */
+  3;
+
+fnNumber =
+  // Comment
+  3;
+
+var fnNumber =
+
+  // Comment
+
+  3;
+
+var fnNumber =
+  // Comment0
+  // Comment1
+  3;
+
+var fnNumber = /* comment */
+  3;
+
+var fnNumber = /* comments0 */
+  /* comments1 */
+  3;
```

Comment:
This test became too long. It's difficult to follow what input gives what output. Could you split it into multiple shorter files, please?